packages/Heading/Heading.jsx

```diff
@@ -1,36 +1,91 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-import safeRest from '../../shared/utils/safeRest'
-import joinClassNames from '../../shared/utils/joinClassNames'
+import styled from 'styled-components'
+import {
+  helveticaNeueLight45,
+  helveticaNeueThin35,
+  helveticaNeueMedium65,
+  wordBreak,
+} from '@tds/shared-typography'
+import { colorWhite, colorText, colorSecondary } from '@tds/core-colours'
+import { media } from '@tds/core-responsive'
 
-import HeadingSup from './HeadingSup/HeadingSup'
+import safeRest from '../../shared/utils/safeRest'
 
-import styles from './Heading.modules.scss'
+import HeadingSup, { StyledHeadingSup } from './HeadingSup/HeadingSup'
 
-const getClassNames = (invert, level) => {
-  if (invert) {
-    return styles.inverted
-  }
-  if (level === 'h1' || level === 'h2') {
-    return styles.secondary
-  }
-  return styles.default
+const HeadingLevels = {
+  h1: {
+    ...helveticaNeueLight45,
+    fontSize: '1.75rem',
+    lineHeight: '1.29', // 36px
+    letterSpacing: '-1.6px',
+    ...media.from('md').css({
+      ...helveticaNeueThin35,
+      fontSize: '2.75rem',
+      lineHeight: '1.18',
+      letterSpacing: '0',
+    }),
+    [`${StyledHeadingSup}`]: {
+      fontSize: '1.25rem',
+      top: '-1em',
+      ...media.from('md').css({ fontSize: '1.25rem', top: '-1.3em' }),
+    },
+  },
+  h2: {
+    ...helveticaNeueLight45,
+    fontSize: '1.5rem',
+    lineHeight: '1.33', // 30px
+    letterSpacing: '-0.7px',
+    ...media.from('md').css({
+      fontSize: '1.75rem',
+      lineHeight: '1.29',
+      letterSpacing: '-0.8px',
+    }),
+    [`${StyledHeadingSup}`]: {
+      fontSize: '1rem',
+      top: '-0.8em',
+      ...media.from('md').css({ fontSize: '1rem', top: '-0.7em' }),
+    },
+  },
+  h3: {
+    ...helveticaNeueMedium65,
+    fontSize: '1.25rem',
+    lineHeight: '1.4', // 28px
+    letterSpacing: '-0.6px',
+    [`${StyledHeadingSup}`]: {
+      fontSize: '0.875rem',
+      top: '-0.5em',
+    },
+  },
+  h4: {
+    ...helveticaNeueMedium65,
+    fontSize: '1rem',
+    lineHeight: '1.25', // 20px
+    letterSpacing: '-0.6px',
+    [`${StyledHeadingSup}`]: {
+      fontSize: '0.875rem',
+      top: '-0.5em',
+    },
+  },
 }
+const StyledHeading = styled.h1(wordBreak, ({ level, invert }) => {
```

Comment:
Can we `export const StyledHeading = ...` instead of `export { StyledHeading }`?